tools/CouchDB_Import_Instruments.php

```diff
@@ -278,7 +286,7 @@ function updateCandidateDocs($Instruments)
      */
     function getInstruments()
     {
-        return \Utility::getAllInstruments();
+        return \NDB_BVL_Instrument::getInstrumentNamesList($this->loris);
```

Comment:
getAllInstruments prints a big "deprecated" warning and tells you to use \NDB_BVL_Instrument::getInstrumentNamesList, so that's what this does.